core/network-libp2p/src/service_task.rs

```diff
@@ -90,11 +73,15 @@ where TProtos: IntoIterator<Item = RegisteredProtocol> {
 		}
 	}
 
-	// Add the bootstrap nodes to the topology and connect to them.
+	/*// Add external addresses.
```

Comment:
Is that still required or just some leftover?

Reply:
Oh, I forgot to uncomment this.